keel-titus-plugin/src/main/kotlin/com/netflix/spinnaker/keel/api/titus/cluster/NoDockerImageFound.kt

```diff
@@ -18,9 +18,10 @@
 package com.netflix.spinnaker.keel.api.titus.cluster
 
 import com.netflix.spinnaker.keel.core.ResourceCurrentlyUnresolvable
+import com.netflix.spinnaker.kork.exceptions.IntegrationException
 
 class NoDigestFound(repository: String, tag: String) :
   ResourceCurrentlyUnresolvable("No digest found for docker image $repository:$tag in any registry")
 
 class RegistryNotFound(titusAccount: String) :
-  ResourceCurrentlyUnresolvable("Unable to find docker registry for titus account $titusAccount")
+  IntegrationException("Unable to find docker registry for titus account $titusAccount")
```

Comment:
This will no longer result in a `ResourceCheckUnresolvable` event.

Reply:
Yes. `IntegrationException` is a sub-class of `SystemException`. For a moment, I was tempted to use just `SystemException` and `UserException` everywhere, but it seemed like a waste to not use descendent types that were more accurate.

Reply:
👍 I'm also in favor of using the more granular exceptions kork has to offer, was mostly just curious to learn what it would look like as an API client. I think sticking with `SYSTEM` for now is fine and later we can decide whether there's usefulness in supporting more granular `origin` values in the events themselves.